### PR TITLE
Login

### PR DESCRIPTION
**Pull request by seromarin**
## Summary
- Login completed with routing

## Screenshot
Captura de Pantalla 2020-11-03 a la(s) 2 47 38 p  m

## Trello Card
https://trello.com/c/ug8AXCwZ/32-login

### CODE REVIEW

File: maquetacion/src/scss/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+@mixin container($width) {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin: 0 auto;
+  width: $width;
+}
+
+@mixin button($color, $bgc, $hover-color, $hover-bgc, $border-color: rgba(0,0,0,0),) {
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_mixins.scss:10]: remove last comma

**Review by idcuadrowolox — CHANGES_REQUESTED** [review]

File: maquetacion/book-detail.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        <div class="nav__content">
+          <img src="./assets/logo-wolox.png" alt="Logo Wolox" class="nav__logo">
+          <a href="/index.html" class="nav__logout">Logout</a>
+        </div>
+      </div>
+    </nav>
+    
+    <div class="container content">
+
+      <div class="back-button">
```
Comment (mcallegari10) [inline maquetacion/book-detail.html:25]: I think this `div` is not needed. If you need the styles, applied it to the `a` tag 😄

File: maquetacion/src/scss/_navbar.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+.nav {
+  background-color: $light;
+  border-top: 6px solid $wolox-blue;
+  box-shadow: 0 4px 10px 0 $half-black;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1;
+  
+  &__content {
```
Comment (mcallegari10) [inline maquetacion/src/scss/_navbar.scss:12]: I think this is not needed, you can use the flexbox attributes in the `.nav` class

File: maquetacion/src/scss/_navbar.scss [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    align-items: center;
+    display: flex;
+    height: 70px;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  &__logo {
+    height: 70px;
+    width: 212.42px;
```
Comment (mcallegari10) [inline maquetacion/src/scss/_navbar.scss:22]: Check the previous PR #8

File: maquetacion/book-list.html [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="./dist/css/application.css" />
+  </head>
+
+  <body>
+
+    <nav id="nav" class="nav">
+      <div class="container">
```
Comment (mcallegari10) [inline maquetacion/book-list.html:15]: Check the CSS, I think this is not needed

File: maquetacion/book-detail.html [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="./dist/css/application.css" />
+  </head>
+
+  <body>
+
+    <nav id="nav" class="nav">
+      <div class="container">
```
Comment (mcallegari10) [inline maquetacion/book-detail.html:15]: Check CSS comment, this is not needed

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**